DOCKER/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM alpine:3.6
 
 # This is the release of tendermint to pull in.
-ENV TM_VERSION 0.10.0
-ENV TM_SHA256SUM a29852b8d51c00db93c87c3d148fa419a047abd38f32b2507a905805131acc19
+ENV TM_VERSION 0.11.0
```

Comment:
lol. one day, when we receive our Greg II from the heavens, we will automate this :P